### ISSUE TITLE
Error: "does not reference a branch" when running Licensed CI

### ISSUE THREAD

**Issue by AnandChowdhary** [jonabc/licensed-ci#44]
Hi @jonabc, thanks for all your hard work with `licensed-ci`!

I tried setting it up in a repository but got this error:

```log
Error: refs/pull/16/merge does not reference a branch
```

This is the PR: https://github.com/koj-co/template/pull/16

Full workflow run: https://github.com/koj-co/template/pull/16/checks?check_run_id=1159826832.

These are the relevant logs:

```log
Run jonabc/licensed-ci@v1
  with:
    config_file: .github/.licensed.yml
    github_token: ***
    user_name: Koj Bot
    user_email: bot@koj.co
    commit_message: :page_facing_up: Update dependency license file
    command: licensed
    workflow: push
    cleanup_on_success: false
git config user.name Koj Bot
git config user.email bot@koj.co
git remote add licensed-ci-origin ***github.com/koj-co/template.git
Error: refs/pull/16/merge does not reference a branch
```

**Comment by jonabc** [comment]
👍  thanks for the report, I'll get this fixed soon.  The underlying cause is that an internal utility only understands how to work with a `GITHUB_REF` environment variable that comes from the `push` event.  I'll get this fixed, but as a short term workaround if you remove `pull_request` from the `on:` options in your workflow file you should still get most of the benefits of licensed-ci.

**Comment by AnandChowdhary** [comment]
Thanks! You're right, it works perfectly without the `pull_request`: https://github.com/koj-co/template/runs/1161028322?check_suite_focus=true.

**Comment by jonabc** [comment]
This should be fixed in v1.5.2, available using either the v1.5.2 or v1 version tags.
